circle.yml

```diff
@@ -4,7 +4,7 @@ machine:
   python:
     version: 2.7.10
   node:
-    version: 0.12.0
+    version: 4.5.0
```

Comment:
It's the one from Ubuntu 16.04 (xenial), which appears to be 4.2.6. Looks like that one is [available on Circle](https://circleci.com/docs/build-image-trusty/#nodejs), so let's use 4.2.6.